frontend/src/pages/org/workflow-detail.ts

```diff
@@ -317,6 +317,13 @@ export class WorkflowDetail extends BtrixElement {
     return this.workflow?.isCrawlRunning && !this.isPaused;
   }
 
+  // Workflow is for a crawl that has failed or canceled
+  private get isFailed() {
```

Comment:
Since there's an opportunity to rename this to be more explicit:
```suggestion
  private get isUnsuccessfullyFinished() {
```